### PR TITLE
shields: mikroe_eth_click: Fix CS pins for LPC55S69

### LINKED ISSUE #31748: boards:lpcxpresso55s69: Manual toggling of CS required with ETH Click shield
When using the SPI interface on the LPC55S69 and the Mikroe ETH Click shield (likely other shields as well), the SPI CS line must be manually toggled as a GPIO pin, requiring the following change to [dts/arm/nxp/nxp_lpc55S6x_common.dtsi](https://github.com/zephyrproject-rtos/zephyr/blob/78c77da578987d7b4774509347d14f9c04eb07da/dts/arm/nxp/nxp_lpc55S6x_common.dtsi#L237-L240):

```
	hs_lspi: spi@9f000 {
		compatible = "nxp,lpc-spi";
		/* Enabling cs-gpios below will allow using GPIO CS,
		 rather than Flexcomm SS */
		cs-gpios = <&gpio0 20 GPIO_ACTIVE_LOW>,
			<&gpio1 1 GPIO_ACTIVE_LOW>,
			<&gpio1 12 GPIO_ACTIVE_LOW>,
			<&gpio1 26 GPIO_ACTIVE_LOW>;
		reg = <0x9f000 0x1000>;
		interrupts = <59 0>;
		clocks = <&syscon MCUX_HS_SPI_CLK>;
		label = "HS_LSPI";
		status = "disabled";
		#address-cells = <1>;
		#size-cells = <0>;
	};
```

**To Reproduce**

Run and flash with the LPCXpresso55S69 (output on USB serial) with and without the .dtsi file changes:

```bash
$ west build -p auto -b lpcxpresso55s69_cpu0 zephyr/samples/net/dhcpv4_client -- -DSHIELD=mikroe_eth_click
$ pyocd flash -t lpc55s69 build/zephyr/zephyr.elf 
```

**Expected behavior**

The manual GPIO selection and file change in this .dtsi file should not be required here.

### PR DESCRIPTION
**Pull request by elcritch**
Fix CS pins for Mikroe Click Ethernet shield overlay on LPC55S69's. Should resolve #31748.

### CODE REVIEW

**Comment by elcritch** [comment]
I ran this patch on a `lpcxpresso55s69` board to verify it works.

File: boards/shields/mikroe_eth_click/boards/lpcxpresso55s69_cpu0.overlay [hunk 1]
```diff
@@ -9,6 +9,11 @@
 &mikrobus_spi {
 	status = "okay";
 
+	cs-gpios = <&gpio0 20 GPIO_ACTIVE_LOW>,
+		<&gpio1 1 GPIO_ACTIVE_LOW>,
+		<&gpio1 12 GPIO_ACTIVE_LOW>,
+		<&gpio1 26 GPIO_ACTIVE_LOW>;
```
Comment (erwango) [inline boards/shields/mikroe_eth_click/boards/lpcxpresso55s69_cpu0.overlay:15]: Could you fix indentation?

File: boards/shields/mikroe_eth_click/boards/lpcxpresso55s69_ns.overlay [hunk 2]
```diff
@@ -9,6 +9,11 @@
 &mikrobus_spi {
 	status = "okay";
 
+	cs-gpios = <&gpio0 20 GPIO_ACTIVE_LOW>,
+		<&gpio1 1 GPIO_ACTIVE_LOW>,
+		<&gpio1 12 GPIO_ACTIVE_LOW>,
+		<&gpio1 26 GPIO_ACTIVE_LOW>;
```
Comment (erwango) [inline boards/shields/mikroe_eth_click/boards/lpcxpresso55s69_ns.overlay:15]: Same here.

**Review by erwango — CHANGES_REQUESTED** [review]
Indentation nitpick, otherwise LGTM.
Thanks for the fix.

**Comment by elcritch** [comment]
Fixed the indentation and the sign-off issue.

File: boards/shields/mikroe_eth_click/boards/lpcxpresso55s69_cpu0.overlay [hunk 3]
```diff
@@ -9,6 +9,11 @@
 &mikrobus_spi {
 	status = "okay";
 
+	cs-gpios = <&gpio0 20 GPIO_ACTIVE_LOW>,
+			   <&gpio1 1 GPIO_ACTIVE_LOW>,
+			   <&gpio1 12 GPIO_ACTIVE_LOW>,
+			   <&gpio1 26 GPIO_ACTIVE_LOW>;
```
Comment (erwango) [inline boards/shields/mikroe_eth_click/boards/lpcxpresso55s69_cpu0.overlay:15]: Looks off again.
Please use 8 chars tab and complete with for fine tuning.

Comment (elcritch) [reply boards/shields/mikroe_eth_click/boards/lpcxpresso55s69_cpu0.overlay:15]: Ok, fixed and followed the coding style guide. Thanks for the heads up.

**Review by erwango — APPROVED** [review]
Thanks for the change. Approved

**Review by MaureenHelm — APPROVED** [review]

**Review by dleach02 — APPROVED** [review]

**Pull request merged**